style/index.js

```diff
@@ -538,7 +546,7 @@ function generateFreemapStyle({
       .rule()
         .textSymbolizer(font().line(500).end({ fill: 'black', size: 11, haloRadius: 1.5, haloOpacity: 0.2, dy: '4 + [off1] * 2.5' }), '[refs1]')
         .textSymbolizer(font().line(500).end({ fill: 'black', size: 11, haloRadius: 1.5, haloOpacity: 0.2, dy: '-4 - [off2] * 4' }), '[refs2]')
-    .style('contours', { opacity: 0.33 })
+    .style('contours', { opacity: 0.8 })
```

Comment:
Mozno by sa s tym dalo pohrat (napr opacity 0.5, alebo dat to trosku hrubsie), ale 0.8 uz je imho dost vyrazne. Minimalne mimo pohori to asi bude posobit rusivo. A aj na strmych kopcoch ked to bude nahusto.

Reply:
Suhlasim. Asi by to trebalo urobit pre rozne zoomy samostatne, toto som dal len tak narychlo, aby som vrstevnice aspon videl.